link/src/main/java/com/stripe/android/link/ui/verification/VerificationViewModel.kt

```diff
@@ -108,7 +108,7 @@ internal class VerificationViewModel @Inject constructor(
 
     fun onBack() {
         clearError()
-        navigator.onBack()
+        navigator.onBack(true)
```

Comment:
Nit: Would be nice to use named parameters for this, so it’s easier to understand what the boolean stands for.

Reply:
Done